### DISCUSSION TITLE
Android example: How was this built?

### DISCUSSION THREAD

**Discussion by rjpikus** [ggml-org/whisper.cpp#751] [Q&A] [answered]
@Digipom how did you build the initial android app project skeleton? Did you use the IDE's pre-made projects or possibly something else? Is there a base project you used?

**Comment by Digipom** [comment, marked as answer]
Hi @rjpikus, I used one of Android Studio's templates and then modified that based on Android's recommended architecture (https://developer.android.com/topic/architecture)

**Reply by rjpikus** [reply]
Thank you, it appears the android studio templates are a step behind https://github.com/android/architecture-templates/tree/multimodule

**Reply by Digipom** [reply]
Interesting, thanks for the link!

**Reply by rjpikus** [reply]
No that's what you sent me, thanks to you.
